muonShieldOptimization/run_reco.py

```
@@ -132,13 +133,15 @@ def executeSimple(prefixes: list[str], reset=False) -> None:
                         print("wait a minute")
                         time.sleep(100)
                 print("launch reco", x)
-                proc[x] = 1
+                proc[x] = (inputfile, geofile)
                 with contextlib.suppress(Exception):
                     os.system("rm logRec")
```

Ruff (0.16.1)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)